### PR TITLE
refactor: drop HANT configuration for WeaselTSF

### PR DESCRIPTION
**Pull request by fxliang**
register both SUBLANG_CHINESE_TRADITIONAL and SUBLANG_CHINESE_SIMPLIFIED in the same DLL, and enable one of them by WeaselSetup, the other can be added to language option in Setting App of Windows manually.

now hant and hans could be available at the same time.
image

in the future, more sublang or langids could be added
if WeaselIME is dopped, HANT configuration can be drop completely

fixed: #704

### CODE REVIEW

File: WeaselSetup/imesetup.cpp [hunk 1]
```diff
@@ -492,6 +495,16 @@ int register_text_service(const std::wstring& tsf_path,
   // if (silent)  // always silent
   { params = L" /s " + params; }
 
+  if (hant) {
```
Comment (determ1ne) [inline WeaselSetup/imesetup.cpp:498]: 这里似乎设置环境变量只是让 DLL 在注册时获取到？这样的话建议不要设置一个永久的环境变量，而是使用 `regsvr32 /i:hans` 与 `regsvr32 /i:hant` 传递参数，这样 `regsvr32.exe` 会调用带有参数的 `DllInstall`/`DllUninstall` 而不是 `DllRegisterServer` 与 `DllUnregisterServer` 。

FYI:
https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/regsvr32
https://learn.microsoft.com/en-us/windows/win32/api/shlwapi/nf-shlwapi-dllinstall?redirectedfrom=MSDN

Comment (fxliang) [reply WeaselSetup/imesetup.cpp:498]: 是这个目的，而且这个是进程有效吧？

https://learn.microsoft.com/zh-CN/windows/win32/api/winbase/nf-winbase-setenvironmentvariable

Comment (determ1ne) [reply WeaselSetup/imesetup.cpp:498]: > 是这个目的，这个是进程有效吧？
> 
> https://learn.microsoft.com/zh-CN/windows/win32/api/winbase/nf-winbase-setenvironmentvariable

看起来是这样的

**Review by determ1ne — APPROVED** [review]

**Comment by fxliang** [comment]
相关还有一点点东西可以再改改

**Pull request merged**